### ISSUE TITLE
User Story: **Cross Validation Search**

### ISSUE THREAD

**Issue by whelew** [whelew/P-Five-Mildew#12]
As a **Data Engineer** I can **perform a cross validation search on my model** so that **I can see which hyperparmeters and model will give the best performance**

**Acceptance Criteria:**

- Add multiple hyperparameters for each layer of the neural network model

- Set cross validation search

- Score different models against a performance metric.
